src/Process/Runner/Parallel/ProcessBearer.php

```diff
@@ -33,36 +33,16 @@
 
 declare(strict_types=1);
 
-namespace Infection\Event\Subscriber;
+namespace Infection\Process\Runner\Parallel;
 
-use Infection\Event\MutantsCreationWasStarted;
-use function Safe\sprintf;
-use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
 
 /**
  * @internal
  */
-final class CiMutantCreatingConsoleLoggerSubscriber implements EventSubscriber
+interface ProcessBearer
 {
-    private $output;
+    public function getProcess(): Process;
 
-    public function __construct(OutputInterface $output)
-    {
-        $this->output = $output;
-    }
-
-    public function getSubscribedEvents(): array
-    {
-        return [
-            MutantsCreationWasStarted::class => [$this, 'onMutantsCreationWasStarted'],
-        ];
-    }
-
-    public function onMutantsCreationWasStarted(MutantsCreationWasStarted $event): void
-    {
-        $this->output->writeln([
-            '',
-            sprintf('Creating mutated files and processes: %s', $event->getMutantCount()),
-        ]);
-    }
+    public function markTimeout(): void;
```

Comment:
`markAsTimedOut`?

Reply:
It was like that. Isn't that going to be unrelated change?

Reply:
Fair I'll propose it in a dedicated PR